cuda_core/cuda/core/experimental/_device.py

```diff
@@ -930,6 +930,291 @@ def multicast_supported(self) -> bool:
+        """
+        bool: True if device supports Vulkan CIG (Compute Interop Graphics), False if not.
+        """
+        return bool(self._get_cached_attribute(driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_VULKAN_CIG_SUPPORTED))
+
+    @property
+    def gpu_pci_device_id(self) -> int:
+        """
+        int: GPU PCI device ID.
+        """
+        return self._get_cached_attribute(driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_GPU_PCI_DEVICE_ID)
+
+    @property
+    def gpu_pci_subsystem_id(self) -> int:
+        """
+        int: GPU PCI subsystem ID.
+        """
+        return self._get_cached_attribute(driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_GPU_PCI_SUBSYSTEM_ID)
+
+    @property
+    def host_numa_virtual_memory_management_supported(self) -> bool:
+        """
+        bool: True if device supports host NUMA virtual memory management, False if not.
+        """
+        return bool(
+            self._get_cached_attribute(
+                driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_HOST_NUMA_VIRTUAL_MEMORY_MANAGEMENT_SUPPORTED
+            )
+        )
+
+    @property
+    def host_numa_memory_pools_supported(self) -> bool:
+        """
+        bool: True if device supports host NUMA memory pools, False if not.
+        """
+        return bool(
+            self._get_cached_attribute(driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_HOST_NUMA_MEMORY_POOLS_SUPPORTED)
+        )
+
+    @property
+    def host_numa_multinode_ipc_supported(self) -> bool:
+        """
+        bool: True if device supports host NUMA multinode IPC, False if not.
+        """
+        return bool(
+            self._get_cached_attribute(driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_HOST_NUMA_MULTINODE_IPC_SUPPORTED)
+        )
+
+    @property
+    def host_memory_pools_supported(self) -> bool:
+        """
+        bool: True if device supports host memory pools, False if not.
+        """
+        return bool(
+            self._get_cached_attribute(driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_HOST_MEMORY_POOLS_SUPPORTED)
+        )
+
+    @property
+    def host_virtual_memory_management_supported(self) -> bool:
+        """
+        bool: True if device supports host virtual memory management, False if not.
+        """
+        return bool(
+            self._get_cached_attribute(
+                driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_HOST_VIRTUAL_MEMORY_MANAGEMENT_SUPPORTED
+            )
+        )
+
+    @property
+    def host_alloc_dma_buf_supported(self) -> bool:
+        """
+        bool: True if device supports host allocation DMA buffer operations, False if not.
+        """
+        return bool(
+            self._get_cached_attribute(driver.CUdevice_attribute.CU_DEVICE_ATTRIBUTE_HOST_ALLOC_DMA_BUF_SUPPORTED)
+        )
+
+    @property
+    def only_partial_host_native_atomic_supported(self) -> bool:
+        """
+        bool: True if device supports only partial host native atomic operations, False if not.
```

Comment:
I'm going to assume that "Partial host native atomic operations" is meaningful to our users :)

Reply:
https://github.com/NVIDIA/cuda-python/pull/1046/commits/faa9ce2e5b76efd7467e41927df7c52ee4ffb663